packages/ns-api/files/ns.flashstart

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/python3
+
+#
+# Copyright (C) 2023 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-2.0-only
+#
+
+import json
+import subprocess
+import sys
+
+from euci import EUci
+from nethsec.utils import ValidationError, validation_error, get_all_by_type, generic_error
+
+
+def __fetch_data(e_uci):
+    return {
+        'enabled': e_uci.get('flashstart', 'global', 'enabled', dtype=bool, default=False),
+        'username': e_uci.get('flashstart', 'global', 'username', default=''),
+        'password': e_uci.get('flashstart', 'global', 'password', default=''),
+        'zones': [zone for zone in e_uci.get('flashstart', 'global', 'zones', list=True, default=[])
+                  if zone != ''],
+        'bypass': [bypass for bypass in e_uci.get('flashstart', 'global', 'bypass', list=True, default=[])
+                   if bypass != '']
+    }
+
+
+cmd = sys.argv[1]
+
+if cmd == 'list':
+    print(json.dumps({
```

Comment:
We need also an api to list the zones. It should return all zones except wan.

Reply:
So it's based off firewall configuration?